docs/getting-started/summaryofchanges.md

```diff
@@ -56,6 +60,21 @@ The following features and enhancements were added.
 - Added a more helpful and actionable description to message ZWEAM511E, which occurs when API ML does not trust the certificate provided by the service. [#818](https://github.com/zowe/api-layer/issues/818)
 
 #### Zowe App Server
+- Added option to write JSON files without using Node.js. [#137](https://github.com/zowe/zlux-app-server/pull/137)
+- Register ZSS to Zowe API ML using a configuration script. This is similar to how the API ML team registers their APIs (for example, the Jobs API). [#208](https://github.com/zowe/zss/pull/208)
```

Comment:
This might be too much info for the user to care.
"ZSS is now automatically registered to the API Mediation Layer when both are present, using a static registration file"